app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "source": "flake8",
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code")
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "path": file_path,
+        "status": "failed" if errors else "passed",
+        "errors": [format_linter_error(i) for i in errors]
```

Comment:
what is "i"?
it's preferable to give understandable names to variables
use "error" instead of "i"

Reply:
sorry, my bad